### PR TITLE
Doc site Improvements

### LINKED ISSUE #1165: Fork Corner for Landing Page
I will add my [Fork Corner](https://github.com/WarenGonzaga/fork-corner) to the homepage.

### PR DESCRIPTION
**Pull request by WarenGonzaga**
Hi @eltonmesquita I add some cool updates.

- up to date copy
- add fork corner to homepage
- capitalize contributor section labels
- update builds

---

Closes #1165

### CODE REVIEW

File: docsSource/template.html [hunk 1]
```diff
@@ -92,7 +96,7 @@ <h2 class="docs-mainTitle">
           </div>
           <aside class="sidebar">
             <div class="meta">
-              <a href="https://github.com/daneden/animate.css" target="_blank" class="icon-github">
+              <a href="https://github.com/animate-css/animate.css" target="_blank" class="icon-github">
```
Comment (eltonmesquita) [inline docsSource/template.html:99]: Nice catch!

File: docs/index.html [hunk 2]
```diff
@@ -1272,7 +1276,7 @@ <h3>Code of Conduct</h3>
           </div>
           <aside class="sidebar">
             <div class="meta">
-              <a href="https://github.com/daneden/animate.css" target="_blank" class="icon-github">
+              <a href="https://github.com/animate-css/animate.css" target="_blank" class="icon-github">
```
Comment (eltonmesquita) [inline docs/index.html:1279]: Nice one.

File: docs/index.html [hunk 3]
```diff
@@ -35,7 +35,11 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.20.0/themes/prism-tomorrow.min.css"
       rel="stylesheet"
     />
-
+    <link 
+      href="https://cdn.jsdelivr.net/npm/fork-corner/dist/fork-corner.min.css"
```
Comment (eltonmesquita) [inline docs/index.html:39]: Is it set as `position: fixed` or `absolute`?

**Review by eltonmesquita — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 3 above)
Comment (WarenGonzaga) [reply docs/index.html:39]: It is set as absolute and the parent is relative in position and I position it on top left side of the homepage. Like this one.
image

**Review by WarenGonzaga — COMMENTED** [review]
All good...

File: docs/index.html (hunk 3 above)
Comment (eltonmesquita) [reply docs/index.html:39]: Can we make it smaller? I think it's a bit big.

Comment (WarenGonzaga) [reply docs/index.html:39]: Yes! the good thing is that this fork-corner I made is the same as animate.css the way you customize it. Just add ``fc-size-small`` let me do the changes.

**Comment by WarenGonzaga** [comment]
It will look like this @eltonmesquita 
image

I did a quick hot fix to my [fork-corner](https://github.com/warengonzaga/fork-corner) because I set it up the body with ``overflow: hidden`` that's why can't scroll down. But now it works correctly.

**Comment by eltonmesquita** [comment]
It's better now. But would you mind to split this fork-corner commit from this PR? It would work better with our git history.

**Comment by WarenGonzaga** [comment]
> It's better now. But would you mind to split this fork-corner commit from this PR? It would work better with our git history.

Ow... remove for now? then create another PR?

**Comment by eltonmesquita** [comment]
Yep. Remove the fork-corner commit and open a new PR (you can use `git cherry-pick` for this and you'll have to `git push --force` to update this PR, but it's ok).

**Comment by WarenGonzaga** [comment]
I'm gonna close this and redo the updates and will open two separate PRs.

**Pull request closed without merging**